app/views/notes/_comment.html.erb

```diff
@@ -118,7 +118,7 @@
   <div style="border: 1px solid #e7e7e7;padding: 18px;" id="c<%= comment.cid %>show">
 
     <% comment_body = comment.render_body %>
-    <p id="comment-body-<%= comment.cid %>"><%= raw filtered_comment_body(comment_body) %></p>
+    <p id="comment-body-<%= comment.cid %>"><%= raw insert_extras(comment_body) %></p>
```

Comment:
Awesome! This is indeed the method we need to add, but do you think you could add it within the `filtered_comment_body` method, and would that work? 

https://github.com/publiclab/plots2/blob/af82259df59b96fa043bbde4bb190278ee560347/app/helpers/application_helper.rb#L120-L125

Basically that method is used to trim out email signatures and such, so we kind of need to run both. Could you put the `insert_extras` method on maybe both lines 122 and 124 of the application_helper.rb file?

Reply:
I could, I'm not totally sure what you would prefer.  I could add `insert_extras` to the `filtered_comment_body` but that may may make uses of `filtered_comment_body` be able to have the the tagging abilities of `insert_extras` which may be overkill.  I just tried wrapping `insert_extras` around `filtered_comment_body` and it worked on my local host fine. 
```
    <p id="comment-body-<%= comment.cid %>"><%= raw insert_extras(filtered_comment_body(comment_body)) %></p>
```
Might be a more feasible limited approach.  If you like that I'll update and push again.